main.py

```diff
@@ -21,7 +21,60 @@ def parse(self, response):
         # open_in_browser(response)
```

Comment:
Might want to move this commented-out line into `parseResults` for folks testing locally

Reply:
Sure! Good call